build-msys.sh

```diff
@@ -0,0 +1,5 @@
+mkdir -p build
+cd build
+cmake -G "MSYS Makefiles" -DCMAKE_INSTALL_PREFIX=C:/MinGW -DCMAKE_BUILD_TYPE=DEBUG ..
+make
+make install
```

Comment:
This is rather just a helper script for myself. We don't need to take that over into the actual repo. I think I'll convert it into an extra paragraph for the README.build.md and remove it then.